Libraries/NativeAnimation/Drivers/RCTSpringAnimation.m

```diff
@@ -0,0 +1,209 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+#import "RCTSpringAnimation.h"
+
+#import <UIKit/UIKit.h>
+
+#import "RCTConvert.h"
+#import "RCTAnimationUtils.h"
+#import "RCTDefines.h"
+#import "RCTValueAnimatedNode.h"
+
+@interface RCTSpringAnimation ()
+
+@property (nonatomic, strong) NSNumber *animationId;
+@property (nonatomic, strong) RCTValueAnimatedNode *valueNode;
+@property (nonatomic, assign) BOOL animationHasBegun;
+@property (nonatomic, assign) BOOL animationHasFinished;
+
+@end
+
+@implementation RCTSpringAnimation
+{
+  NSArray<NSNumber *> *_frames;
+  CGFloat _toValue;
+  CGFloat _fromValue;
+  BOOL _overshootClamping;
+  CGFloat _restDisplacementThreshold;
+  CGFloat _restSpeedThreshold;
+  CGFloat _tension;
+  CGFloat _friction;
+  CGFloat _initialVelocity;
+  NSTimeInterval _animationStartTime;
+  NSTimeInterval _animationCurrentTime;
+  RCTResponseSenderBlock _callback;
+
+  CGFloat _lastPosition;
+  CGFloat _lastVelocity;
+  CGFloat _outputValue;
```

Comment:
I think we can get rid of this, we're only writing to it and never using it's value.